### ISSUE TITLE
Regarding addition of old and new webinar links

### ISSUE THREAD

**Issue by Jassi10000-zz** [EddieHubCommunity/Hackathons#1]
@eddiejaoude Pls also include links of your old and new webinars that you usually took during Hackathons , it can be easy for many new developers to gain an insight ( this is a request from some of my friends who are new to hackathons and great fan of yours ofcourse)

**Comment by eddiejaoude** [comment]
Hi @Jassi10000 , sorry for my slow reply. Can you explain a bit more about what resources would be useful?

**Comment by Jassi10000-zz** [comment]
@eddiejaoude  No issues sir , I know you have other stuff to handle too
 yeah ,  leme just make it simple , 
There are some of my friends who are beginners and want to take part in hackathons , but don't know how to actually approach them and do some stuff while participating 
I suggested them your channel and this repository , they liked your videos and material , but the thing is they wish that Eddie would also have videos of how to perform in great hackathons like those organized in MLH and some other big organizations like DevFolio and Devpost 

So basically they are demanding some videos and links for guidance regarding the same ,  and if you wish to do this , pls do it for beginners and well as for people who are above beginner level

**Comment by eddiejaoude** [comment]
Sure. Can you ask them to write a list of topics to cover here and I can use that to create the video and cover the topics

**Comment by Jassi10000-zz** [comment]
Sure, I will collect the topics and share it with you 
Thanks .

**Comment by adityaraute** [comment]
What happened here? Any progress?

**Comment by Jassi10000-zz** [comment]
Actually I shared the data for same to Eddie on Discord 
Anyways I'll tell them here too  : 
Topics : 
* Git and Github webinars (including info about Github Pages )
* DevOps ( with GitOps)
* ML webinars 
* Webinars on MERN Stack 
* Webinars on Cloud App development

**Comment by eddiejaoude** [comment]
This might be better as a GitHub Discussion?
